package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentstack/apps-cli",
-  "version": "1.0.7",
+  "version": "1.0.8",
```

Comment:
@cs-raj , version bump is not required. We will perform minor version bump after @harshithad0703  completes the reinstall command.

Reply:
Got it, I will revert it